core/src/test/java/org/frankframework/management/bus/endpoints/TestFileViewer.java

```diff
@@ -0,0 +1,131 @@
+package org.frankframework.management.bus.endpoints;
+
+import org.frankframework.management.bus.BusAction;
+import org.frankframework.management.bus.BusException;
+import org.frankframework.management.bus.BusMessageUtils;
+import org.frankframework.management.bus.BusTestBase;
+import org.frankframework.management.bus.BusTopic;
+import org.frankframework.management.bus.ResponseMessageBase;
+import org.frankframework.testutil.TestFileUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.MediaType;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHandlingException;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import java.io.InputStream;
+
+public class TestFileViewer extends BusTestBase {
+
+	private static final String TestFileName = "FileViewer.txt";
+	private static final String TestFilePath = "/Management/" + TestFileName;
+
+	@Test
+	@WithMockUser(authorities = { "ROLE_IbisTester" })
+	public void getFileContent() throws Exception {
+		MessageBuilder<String> request = createRequestMessage("NONE", BusTopic.FILE_VIEWER, BusAction.GET);
+		request.setHeader("resultType", "plain");
+		request.setHeader("fileName", TestFileUtils.getTestFilePath(TestFilePath));
+
+		Message<?> response = callSyncGateway(request);
+		String expectedTxt = TestFileUtils.getTestFile(TestFilePath);
+		String result = new String(((InputStream) response.getPayload()).readAllBytes());
```

Comment:
Ik denk dat jij de eerste bent die readAllBytes gebruikt! Toch is het misschien handiger om `StreamUtil.streamToString` te gebruiken omdat die kijkt naar de charset van het bestand. Zie:  https://github.com/frankframework/frankframework/blob/master/commons/src/main/java/org/frankframework/util/StreamUtil.java#L437